include/uv.h

```diff
@@ -1120,6 +1131,8 @@ UV_EXTERN int uv_get_process_title(char* buffer, size_t size);
 UV_EXTERN int uv_set_process_title(const char* title);
 UV_EXTERN int uv_resident_set_memory(size_t* rss);
 UV_EXTERN int uv_uptime(double* uptime);
+UV_EXTERN uv_os_fd_t uv_get_osfhandle(int fd);
+UV_EXTERN int uv_open_osfhandle(uv_os_fd_t os_fd);
```

Comment:
These two functions aren't applicable to libuv master, as the issue they work around was fixed by #1166 (note  the compilation warnings that are being caused by their presence)

Reply:
There might, yes. I don’t think there’s much reason to include it in libuv though, since it’s so simple. If you want to include it though, it should be moved into this header (static inline like uv_get_osfhandle, or `#define uv_open_osfhandle _open_osfhandle`) so that the caller will be certain to get the fd from the caller’s libc table.

Reply:
Ok, thanks!

@bzoz thoughts on whether we should keep `uv_get_osfhandle()` on master?

Reply:
IMHO we should drop it

Reply:
@bzoz sorry, I meant `uv_open_osfhandle()`